add Fdroid repo

i'm fine with Play Store, but you would certainly gain users there

```
  - versionName: 1.6.0
    versionCode: 1060000
    commit: e6423dfba9f7b5b85c216256c65dfe10278a808a
    subdir: app
    gradle:
      - free
    prebuild:
      - sed -i -e '/play-/d' -e '/firebase-bom/,+7d' -e '/dynamic-features-fragment/d'
        build.gradle.kts
      - sed -i -e '/maven {/,+4d' ../settings.gradle.kts
      - sed -i -e 's/versionCode = .*/versionCode = "$$VERCODE$$"/' -e 's/changeset
        = .*/changeset = "unknown"/' ../buildSrc/src/main/kotlin/RepositoryChangeset.kt
```

So we remove some "usual suspects", we remove your "custom repo" so then it fails with:
```
> Task :app:processFreeReleaseResources FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processFreeReleaseResources'.
> A failure occurred while executing com.android.build.gradle.internal.res.LinkApplicationAndroidResourcesTask$TaskAction
   > Android resource linking failed
     ERROR:/home/vagrant/build/com.geekorum.ttrss.free/app/src/main/res/navigation/add_feed_installer.xml:40: AAPT: error: attribute moduleName (aka com.geekorum.ttrss.free:moduleName) not found.
         
     ERROR:/home/vagrant/build/com.geekorum.ttrss.free/app/src/main/res/navigation/add_feed_installer.xml:51: AAPT: error: attribute moduleName (aka com.geekorum.ttrss.free:moduleName) not found.
         
     ERROR:/home/vagrant/build/com.geekorum.ttrss.free/app/src/main/res/navigation/articles_list.xml:112: AAPT: error: attribute moduleName (aka com.geekorum.ttrss.free:moduleName) not found.
         
     ERROR:/home/vagrant/build/com.geekorum.ttrss.free/app/src/main/res/xml/pref_general.xml:67: AAPT: error: resource string/oss_license_title (aka com.geekorum.ttrss.free:string/oss_license_title) not found.
```

...as expected right? Per https://github.com/fbarthelery/geekttrss/blob/master/settings.gradle.kts#L45-L48

We also hardcode versionname and versioncode as `hg` (mercurial) will fail (error code 255) somewhere around here https://github.com/fbarthelery/geekttrss/blob/5b1dc61ca34d737611d58e8cd5c66729492acbd1/buildSrc/src/main/kotlin/RepositoryChangeset.kt

Well... 
```
FAILURE: Build failed with an exception.

* Where:
Build file '/home/vagrant/build/com.geekorum.ttrss.free/app/build.gradle.kts' line: 27

* What went wrong:
Plugin [id: 'com.google.android.gms.oss-licenses-plugin', version: '0.10.6.1', artifact: 'com.google.android.gms:oss-licenses-plugin:0.10.6.1'] was not found in any of the following sources:

- Gradle Core Plugins (plugin is not in 'org.gradle' namespace)
- Plugin Repositories (could not resolve plugin artifact 'com.google.android.gms:oss-licenses-plugin:0.10.6.1')
  Searched in the following repositories:
    Gradle Central Plugin Repository
    MavenRepo
    Google
```

...the list of trusted repos is here: https://gitlab.com/fdroid/fdroidserver/-/blob/2a60f6c195dfd24149617ceb1df1731fb3bf50c8/fdroidserver/scanner.py#L452-L47 and your github repo is not allowed.

--- a/README.md
+++ b/README.md
@@ -21,7 +21,7 @@ The project builds in 2 flavors :
    * The Google flavor is distributed on the [Google Play store](https://play.google.com/store/apps/details?id=com.geekorum.ttrss).
      It uses Crashlytics and other Google Play services to retrieve crash reports.
 
-   * The free flavor is not distributed but can be build from the sources. It doesn't contains any Google Play services.
+   * The free/libre flavor is distributed on [F-Droid](https://f-droid.org/packages/com.geekorum.ttrss.free/), and can also be built from the sources. It doesn't contains any Google Play services. The APK of the latest version of this flavor can also be downloaded from the [Releases Section](https://github.com/fbarthelery/geekttrss/releases/latest).
 
 
 Build instructions

